BookingsAPI/Bookings.DAL/Commands/HearingService.cs

```diff
@@ -88,6 +114,39 @@ public async Task UpdateHearingCaseName(Guid hearingId, string caseName)
             await _context.SaveChangesAsync();
         }
 
+        public Task<List<LinkedParticipant>> CreateParticipantLinks(List<Participant> participants, List<LinkedParticipantDto> linkedParticipantDtos)
```

Comment:
If your creating links, you should be manipulating the video hearing object rather than creating an independent list and returning it